os/android/libhwcservice/hwcserviceapi.cpp

```diff
@@ -221,6 +221,46 @@ status_t HwcService_DisplayMode_SetMode(HWCSHANDLE hwcs, uint32_t display,
   return pContext->mControls->DisplayModeSetMode(display, config);
 }
 
+status_t HwcService_Video_EnableHDCPSession_ForDisplay(HWCSHANDLE hwcs,
+                                                       uint32_t display,
+                                                       bool fallback) {
+  HwcsContext* pContext = static_cast<HwcsContext*>(hwcs);
+  if (!pContext) {
+    return -1;
```

Comment:
Given that we are using status_t return type, we ought to use a real return code. I'd suggest `BAD_VALUE`.

Reply:
ackowledeged, however I would like to keep it consistent with other API and do all clean up once. Added task here, if anyone wants to try this out: https://github.com/intel/IA-Hardware-Composer/issues/268